Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2162,6 +2168,25 @@ def get_store_data(service):
         yield store.data.query(**query)
 
 
+class MessagesHandler:
```

Comment:
why do you need a class for it? why can't it just be a regular function that gets the service as an argument?

Reply:
we discussed it and decided that it’s OK in this situation.